tools/build.js

```diff
@@ -15,7 +15,7 @@ const libRoot = path.join(__dirname, '../lib');
 const distRoot = path.join(libRoot, 'dist');
 const esRoot = path.join(libRoot, 'es');
 
-const clean = () => fse.existsSync(libRoot) && fse.remove(libRoot);
+const clean = async () => fse.existsSync(libRoot) && fse.remove(libRoot);
```

Comment:
seems sort of weird... can we just use `removeSync` here and remove the `.then` below?

Reply:
It's also possible. 👍

Reply:
Let me know if you want. Then I'll fix it.

Reply:
let's just do `removeSync` here and remove the `.then` below